### PR TITLE
Remove code-view class from diff view

### LINKED ISSUE #13010: code-view active class being applied to PR code comment editor
What should be the styling for selected lines of code in file view (I think?) is also affecting the code editor because it ends up with the same class structure of .code-view .active

Screen Shot 2020-10-01 at 11 14 18 PM

### PR DESCRIPTION
**Pull request by mrsdizzie**
code-view class seems unecessary here as everything needed style wise comes from various diff classes. This allows comments and comment editor to be styled properly and fixes linked bug.

Fixes #13010

Before:
Screen Shot 2020-10-01 at 11 14 18 PM

After:
Screen Shot 2020-10-01 at 11 57 35 PM

### CODE REVIEW

**Review by zeripath — APPROVED** [review]
Just double check that this is correct on arc green but otherwise lgtm

**Review by CirnoT — APPROVED** [review]
Textarea is still using wrong font, otherwise looks good.

File: web_src/less/_base.less [hunk 1]
```diff
@@ -997,6 +997,7 @@ i.icon.centerlock {
   color: rgba(27, 31, 35, .3);
   width: 1%;
   user-select: none;
+  font-family: @monospaced-fonts, monospace;
```
Comment (bagasme) [inline web_src/less/_base.less:1000]: Why did you add this?

**Comment by mrsdizzie** [comment]
> Textarea is still using wrong font, otherwise looks good.

SimpleMDE seems to use monospace font in all instances (which I don't really like either).
